std/typecons.d

```diff
@@ -1204,6 +1204,11 @@ if (distinctFieldNames!(Specs))
         /**
          * Formats `Tuple` with either `%s`, `%(inner%)` or `%(inner%|sep%)`.
          *
+         * Params:
+         *     writer = A `char` accepting
```

Comment:
Ehm that will be confusing to people because sink and writer are defined terms.
- sink = lambda output
- writer = output range

FYI: Since 2.079.0 toString supports OutputRange writers -> https://dlang.org/changelog/2.079.0.html#toString

tl;dr: keep the name

Reply:
So, not change to output range `toString`, right?

Reply:
Oh you can definitely add a `toString` overload that uses a writer output range, but just renaming the variable might be very confusing.